pwnlib/tubes/process.py

```diff
@@ -876,6 +880,34 @@ def libs(self):
 
         return maps
 
+    def maps(self):
```

Comment:
Use `self.corefile.mappings` or `self.corefile.maps`

Reply:
I have parsed the maps and transform them in corefile.Mappings in c3d246a, since creating a corefile to inspect the maps is annoying a some information is lost, like the name of the heap map.